lib/graphiti/sideload.rb

```diff
@@ -42,6 +42,14 @@ def initialize(name, opts)
       @group_name            = opts[:group_name]
       @polymorphic_child     = opts[:polymorphic_child]
       @parent                = opts[:parent]
+
+      @linkage_always        = if opts[:linkage_always].nil?
```

Comment:
yeah, I don't love the `linkage_always` name either. Agreed on better clarity with your suggestion, but having `include` in there might be confusing since this doesn't have anything to do with the `included` payload, but the rather the relationship.

what about `resource_identifiers = :always` vs `resource_identifiers = :auto`?

Reply:
I don't love the non-boolean, but it is descriptive. Maybe `:when_included` instead of `:auto`

Reply:
nah, you're right. boolean and `always_` is most descriptive. I went with `always_include_resource_ids`